src/hooks/queries/useProductsQuery.ts

```diff
@@ -1,24 +1,40 @@
 import type { UseQueryResult } from '@tanstack/react-query'
-import { useQuery } from '@tanstack/react-query'
+import { useQuery, useQueryClient } from '@tanstack/react-query'
```

Comment:
I can see that useQueryClient has been added to this line, what is the purpose of this and what job will it execute? @GergoN02

Reply:
fyi https://tanstack.com/query/v4/docs/react/reference/useQueryClient
May be good to list out the packages that are being used along with the documentation site for visibility over any package questions that have been introduced. There is a spec issue #1 so a comment in there would be suitable. From my understanding, we are using: react, material UI, TanStack Query and Typescript.

Reply:
README lists the chosen technologies but it is not up to date since it refers to the use of tailwind when we have switched to material UI. A revision to that section would be sufficient.

Reply:
We're using the entire Tanstack Ecosystem of packages, in this case, useQuery is responsible for making the actual API queries, while useQueryClient is responsible for handling updates to the local cache. We are caching a lot of the requests as it helps us optimize query speeds, load times, etc...